hr_employee_id/views/hr_employee_views.xml

```diff
@@ -6,13 +6,12 @@
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="groups_id" eval="[(6, 0, [ref('hr.group_hr_user') ])]"/>
         <field name="arch" type="xml">
-            <field name="identification_id" position="replace"/>
-            <xpath expr="//div[hasclass('oe_title')]" position="inside">
-                <label for="identification_id" class="oe_edit_only"/>
-                <h3>
-                    <field name="identification_id"/>
-                </h3>
-            </xpath>
+            <field name="category_ids" position="after">
+                <field name="identification_id" position="move"/>
```

Comment:
```suggestion
                <h3>
                    <field name="identification_id" placeholder="Identification Number"/>
                </h3>
```
What is position move?
I found when select tags. identification_id will show next to tags.

Reply:
Right! Change done! Could you check again?
I used `position="move"` to avoid the use of the `replace` (that was used before).

Reply:
@astirpe it's okay but why are you switch position after -> before?

Reply:
Just for the look&feel, looks better to me. If you prefer to have it after, just let me know. Doesn't really matter to me.

Reply:
Kanban view show tags -> id
Form view show  id -> tags
it should be similar. but I'm okay :)